css/pages/index.css

```diff
@@ -2,7 +2,180 @@
   box-sizing: border-box;
 }
 
-.container {
+/* Arnab's code */
+body {
+  margin: 0;
+}
+.first-screen {
+  background-color: #041187;
+  width: 100%;
+}
+.first-screen header {
+  position: relative;
+  height: 30%;
+  display: flex;
+  flex-flow: row wrap;
+  justify-content: space-between;
+}
+#logo {
+  width: 5em;
+  height: 5em;
+  position: relative;
+  top: 1.5em;
+  left: 3em;
+}
+.main-nav {
+  width: 44%;
+  height: 1.6em;
+  position: relative;
+  top: 1.3em;
+  display: flex;
+  align-self: center;
+  font-family: 'roboto';
+  justify-content: space-between;
+  flex-wrap: nowrap;
+}
+.nav-text {
+  color: #fff;
+  text-decoration: none;
+  font-size: 1.08em;
+}
+a.nav-text.welcome {
+  color: #88d870;
+  border-bottom: 0.12em solid #88d870;
+}
+.sign-in-button {
+  width: 16em;
+  height: 2.5em;
+  text-align: center;
+  background-color: #041187;
+  color: #fff;
+  border: 0.2em solid #fff;
+  border-radius: 0.6em;
+  position: relative;
+  top: 2.8em;
+  right: 3em;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+#gitButtonLogo {
+  width: 1.5em;
+  height: 1.7em;
+  position: relative;
+  left: 0.5em;
+  border-radius: 20%;
+}
+div.heading-container {
+  flex-direction: column;
+  display: inline-block;
+  position: relative;
+  top: 5em;
+  left: 2em;
+}
+#first-line {
+  font-size: 4.3em;
+  line-height: -3em;
+  width: 9em;
+  height: 1em;
+  display: inline-block;
+  position: relative;
+  top: 0.8em;
+  left: 0.8em;
+}
+#welcome-text {
+  font-family: 'raleway';
```

Comment:
Is there a way we can wrap a div with a font family instead of repetition

Reply:
yes, there is definitely, and I corrected it as well!